### PR TITLE
2.11.0

### PR DESCRIPTION
**Pull request by wmonk**
This is the merged upstream of facebook 1.0.17

### CODE REVIEW

File: packages/react-dev-utils/WebpackDevServerUtils.js [hunk 1]
```diff
@@ -187,7 +183,7 @@ function createCompiler(webpack, config, appName, urls, useYarn) {
       );
       console.log(
         'To ignore, add ' +
-          chalk.cyan('// tslint:disable-next-line') +
+          chalk.cyan('// eslint-disable-next-line') +
```
Comment (DorianGrey) [inline packages/react-dev-utils/WebpackDevServerUtils.js]: This should remain `tslint:disable-next-line` - we're not using ESLint.

File: packages/react-scripts/scripts/utils/createJestConfig.js [hunk 2]
```diff
@@ -34,17 +31,17 @@ module.exports = (resolve, rootDir) => {
...
       '^.+\\.tsx?$': resolve('config/jest/typescriptTransform.js'),
-      '^(?!.*\\.(js|jsx|css|json)$)': resolve('config/jest/fileTransform.js'),
+      '^.+\\.css$': resolve('config/jest/cssTransform.js'),
+      '^(?!.*\\.(js|jsx|mjs|css|json)$)': resolve(
+        'config/jest/fileTransform.js'
+      ),
     },
-    transformIgnorePatterns: [
-      '[/\\\\]node_modules[/\\\\].+\\.(js|jsx|ts|tsx)$',
-    ],
```
Comment (DorianGrey) [inline packages/react-scripts/scripts/utils/createJestConfig.js]: Suppose this pattern should not be removed, but only extended to also cover `mjs`:
```
transformIgnorePatterns: [
    '[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|ts|tsx)$'
]
```

File: packages/react-dev-utils/WebpackDevServerUtils.js (hunk 1 above)
Comment (wmonk) [reply packages/react-dev-utils/WebpackDevServerUtils.js]: I think this is an unneeded change isn't it, as we still use the published `react-dev-utils` not a ts specific one?

Comment (DorianGrey) [reply packages/react-dev-utils/WebpackDevServerUtils.js]: Oh - yeah, my bad,

File: packages/react-scripts/scripts/utils/createJestConfig.js [hunk 3]
```diff
@@ -34,9 +31,11 @@ module.exports = (resolve, rootDir) => {
...
-      '^.+\\.css$': resolve('config/jest/cssTransform.js'),
       '^.+\\.tsx?$': resolve('config/jest/typescriptTransform.js'),
-      '^(?!.*\\.(js|jsx|css|json)$)': resolve('config/jest/fileTransform.js'),
+      '^.+\\.css$': resolve('config/jest/cssTransform.js'),
+      '^(?!.*\\.(js|jsx|mjs|css|json)$)': resolve(
+        'config/jest/fileTransform.js'
+      ),
     },
     transformIgnorePatterns: [
       '[/\\\\]node_modules[/\\\\].+\\.(js|jsx|ts|tsx)$',
```
Comment (DorianGrey) [inline packages/react-scripts/scripts/utils/createJestConfig.js]: As mentioned in the last comment, the pattern should include `mjs`:
```
'[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|ts|tsx)$'
```
This pattern was extended to include `mjs` a couple of months ago already.

Comment (wmonk) [reply packages/react-scripts/scripts/utils/createJestConfig.js]: doh

**Pull request merged**